src/Patient/Cards/PortalCard.php

```diff
@@ -78,6 +85,7 @@ private function setOpts()
             'identifier' => self::CARD_ID,
             'title' => xl('Patient Portal') . ' / ' . xl('API Access'),
             'templateVariables' => [
+                'allowpp' => xl('Allow Patient Portal in Demographics ' . $groupName),
```

Comment:
can't include leading/trailing spacing in translations and need to translate the group title with a special translation function:
```
'allowpp' => xl('Allow Patient Portal in Demographics' . ' ' . xl_layout_label($groupName)),
```

Reply:
actually, would just pass the $groupName into the twig template:
```
'allowpp' => $groupName,
```

Reply:
thanks @bradymiller - i'm doing your first suggestion as it needs the complete string  'Allow Patient Portal in Demographics <groupName>' - rather than simply '<groupName>'